lte/gateway/c/session_manager/test/CMakeLists.txt

```diff
@@ -15,10 +15,10 @@ add_library(SESSIOND_TEST_LIB
 
 target_link_libraries(SESSIOND_TEST_LIB SESSION_MANAGER gmock_main pthread rt)
 
-foreach (session_test session_credit local_enforcer cloud_reporter
-    session_manager_handler sessiond_integ session_state
-    session_store store_client stored_state proxy_responder_handler
-    metering_reporter local_enforcer_wallet_exhaust charging_grant
+foreach (session_test session_credit cloud_reporter
```

Comment:
Please revert these changes

Reply:
Need to fix couple of  unit test errors, deferring for now.

Reply:
I ll resolve it later.

Reply:
@gurrapualt 
It looks like we're still missing: 
`local_enforcer`, `proxy_responder_handler`, `local_enforcer_wallet_exhaust`, `session_manager_handler`, `sessiond_integ` ?

Reply:
Fixed, resolved.